.github/workflows/coverity.yml

```diff
@@ -0,0 +1,23 @@
+name: Coverity
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - main
+
+jobs:
+  lint:
+    name: "Coverity scan"
+    runs-on: ubuntu-latest
+    env: 
+      - COVERITY_SCAN_PROJECT_NAME: ${{ secret.COVERITY_SCAN_PROJECT_NAME }}
+      - COVERITY_SCAN_TOKEN: ${{ secret.COVERITY_SCAN_TOKEN }}
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Run coverify scan script
+        run: ./tools/coverity-scan.sh
```

Comment:
Missing newline at EOF

Reply:
Fixed

Reply:
Nope, it's not fixed. See the red circle icon just above the comments.